metabase/metabase
# ECharts combined scalars chart

Closes https://github.com/metabase/metabase/issues/35815
### Description

This PR implements combined scalars chart which we render when multiple scalars are combined on a dashcard as a regular BarChart.

The drills are not working correctly and they will be addressed further.

### How to verify

1. Create two or more "Number" visualization
2. Add one on a dashboard, then combine others with it
3. Ensure it renders correct bar chart with legend and tooltip

### Demo

<img width="632" alt="Screenshot 2024-01-09 at 11 22 27 PM" src="https://github.com/metabase/metabase/assets/14301985/64b2b74b-c810-4480-87df-0427fc374555">

### Checklist
